agent/consul/acl_client.go

```diff
@@ -94,6 +94,13 @@ func (c *Client) ResolveToken(token string) (acl.Authorizer, error) {
 	return c.acls.ResolveToken(token)
 }
 
+func (c *Client) ResolveTokenToIdentity(token string) (structs.ACLIdentity, error) {
+	// not using ResolveTokenToIdentityAndAuthorizer because in this case we don't
+	// need to resolve the roles, policies and namespace but just want the identity
+	// information such as accessor id.
+	return c.acls.ResolveTokenToIdentity(token)
```

Comment:
Eventually I think it would make sense for the Agent and Server/Client to share an `ACLResolver` similar to how they share the `TLSConfigurator`. It would get rid of the need for all the passthrough funcs. For now though I tried to keep the changes smaller.